cmd/ike/session/session_test.go

```diff
@@ -0,0 +1,40 @@
+package session_test
+
+import (
+	"github.com/aslakknutsen/istio-workspace/cmd/ike/session"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Session operations", func() {
+
+	Context("route parsing", func() {
```

Comment:
This actually would be awesome to test using something like [Gopter](https://github.com/leanovate/gopter) (property based testing). I will take a look in my spare time.

Reply:
Good idea.

Reply:
#100